getting-started/hubspot.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: HubSpot
+description: Implementing Edgee in front of HubSpot for better performance, flexibility, and domain control
+icon: /images/hubspot/hubspot.svg
+---
+
+import EdgeeSdk from '/snippets/edgee-sdk.mdx';
+
+<EdgeeSdk />
+
+Edgee can sit in front of your HubSpot CMS as a proxy to **boost performance**, **strengthen security**, and **seamless integration**.
+
+With this setup, HubSpot-hosted content (such as a blog) can appear under your **main site domain** (e.g., `www.example.com/blog`) while still being served from HubSpot.  
+
+<Note>
+For HubSpot’s full technical documentation, see [Use a reverse proxy with HubSpot](https://developers.hubspot.com/docs/guides/cms/content/performance/reverse-proxy-support).
+</Note>
+
+---
+
+## Edgee Setup Overview
+
+Integrating Edgee with HubSpot involves four main steps:
+
+1. **Configure Edgee** – point Edgee to your HubSpot origin URL (e.g., `https://123456.group15.sites.hubspot.net`).  
+2. **Create Secret Headers in Edgee** – forward headers so HubSpot recognizes requests correctly.  
+3. **Connect or update your domain in HubSpot** – register the domain and disable HubSpot’s built-in CDN.
+4. **Update your DNS to route through Edgee** – ensure all traffic flows to Edgee first, then securely to HubSpot.
+
+
+
+---
+
+### Step 1 — Connect Your Domain to Edgee
+
+<Note>See [Custom Domains](/getting-started/domains) for detailed instructions on configuring domains with Edgee.</Note>
+
+1. Log in to your **Edgee** dashboard.  
+2. Go to **Project Settings → Routing**.  
+3. Click **Create New Route** and configure the following:
+   - Configure **Origin URL**, **Host Override**, **SNI**, and **Cert Hostname** with the same `https://<HubID>.<group>.sites.hubspot.net` domain (e.g. `nhttps://123456.group15.sites.hubspot.net`).
+4. Save and activate your configuration.  
+
+---
+
+### Step 2 — Create Secret Headers in Edgee
+
+1. In Edgee, go to **Security → Secret Headers**.  
+2. Add the following headers so HubSpot accepts and processes requests correctly:  
+
+- **Header name** → `X-HS-Public-Host`  
+  **Header value** → `www.mydomain.com` (the domain pointing to your site)  
+
+- **Header name** → `X-HubSpot-Trust-Forwarded-For`  
+  **Header value** → `true`  
+
+3. Save the header configuration.  
+
+   <img src="/images/hubspot/hubspotheaders.png" />
+
+
+---
+
+### Step 3 — Connect the Domain in HubSpot
+
+1. In HubSpot, go to **Settings → Content → Domains & URLs**.  
+2. Click **Connect a domain**.  
+3. Choose the type (Primary, Secondary, or Redirect) and select which content types you’ll host.  
+4. **Toggle OFF** the option:  
```

Comment:
option name/title is missing here :) I understand there's a screenshot below, but I'd still mention the option name in text for clarity